sh/tests/solutions/array-selector.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash 
```

Comment:
this file is not executable

Reply:
I will change this just to avoid weird behaviour, but it should not be a problem since the test script is running `bash <filename>`